public/externalLibs/sound/soundToneMatrix.js

```diff
@@ -417,7 +417,8 @@ function letter_name_to_frequency(note) {
 }
 
 function midi_note_to_frequency(note) {
-  return 8.1757989156 * Math.pow(2, (note / 12));
+  // Frequency of midi note 0 / C-1.
+  return 8.17579891564371 * Math.pow(2, (note / 12));
```

Comment:
Give a name to this value and declare it at a prominent place: something like
const frequency_of_c0 = ...;

Reply:
Realized that declaring it broke the library.
will discuss this further offline